docs: improve wording in Github Actions plugin README

bhoomiiee opened the pull request:
Improves capitalization and wording in the Github Actions plugin README.
No functional changes.

awanlin reviewed (changes requested):
Thanks for the corrections @bhoomiiee, left one comment and you'll need to add a changeset. The `README` gets published to NPM so we will want it updated there as well. More details on how to do that here: https://github.com/backstage/community-plugins/blob/main/CONTRIBUTING.md#creating-changesets

bhoomiiee commented:
> Thanks for the corrections @bhoomiiee, left one comment and you'll need to add a changeset. The `README` gets published to NPM so we will want it updated there as well. More details on how to do that here: https://github.com/backstage/community-plugins/blob/main/CONTRIBUTING.md#creating-changesets

Thanks for the clarification!
I've added a changeset as suggested and updated the PR. Please let me know if anything else is needed.

awanlin reviewed (changes requested):
Thanks for trying to add the changeset @bhoomiiee, but it looks like you did it in the root of the repo and not in the `github-actions` workspace where you want to run it. Can you remove the two files you added and try again, please?

awanlin commented:
Just noting there are still unresolved comments and there's an error in the CI related to the DCO that needs to be fixed 👍
